apps/mobile/src/theme/textVariants.ts

```diff
@@ -0,0 +1,104 @@
+const heading01 = {
+  fontFamily: 'DiatypeRegular',
+  fontSize: 53,
+  fontStyle: 'normal',
+  fontWeight: 500,
+  lineHeight: 60,
+};
+const heading02 = {
+  fontFamily: 'MarcheSuperPro',
+  fontSize: 44,
+  fontStyle: 'normal',
+  fontWeight: 800,
+  lineHeight: 44,
+  textTransform: 'uppercase',
+};
+const heading03 = {
+  fontFamily: 'MarcheSuperPro',
+  fontSize: 32,
+  fontStyle: 'normal',
+  fontWeight: 800,
+  lineHeight: 35,
+  letterSpacing: 0.64,
+  textTransform: 'uppercase',
+};
+const heading04 = {
+  fontFamily: 'DiatypeRegular',
+  fontSize: 26,
+  fontStyle: 'normal',
+  fontWeight: 500,
+  lineHeight: 36,
+};
+const heading05 = {
+  fontFamily: 'DiatypeRegular',
+  fontSize: 24,
+  fontStyle: 'normal',
+  fontWeight: 500,
+  lineHeight: 28,
+};
+const label01 = {
+  fontFamily: 'DiatypeRegular',
+  fontSize: 16,
+  fontStyle: 'normal',
+  fontWeight: 500,
+  lineHeight: 24,
+  letterSpacing: 0.16,
+};
+const label02 = {
+  fontFamily: 'DiatypeRegular',
+  fontSize: 15,
+  fontStyle: 'normal',
+  fontWeight: 500,
+  lineHeight: 20,
+};
+const label03 = {
+  fontFamily: 'DiatypeRegular',
+  fontSize: 13,
+  fontStyle: 'normal',
+  fontWeight: 500,
+  lineHeight: 16,
+};
+const body01 = {
+  fontFamily: 'DiatypeRegular',
+  fontSize: 17,
+  fontStyle: 'normal',
+  fontWeight: 400,
+  lineHeight: 24,
+};
+const body02 = {
+  fontFamily: 'DiatypeRegular',
+  fontSize: 15,
+  fontStyle: 'normal',
+  fontWeight: 400,
+  lineHeight: 20,
+};
+const caption01 = {
+  fontFamily: 'DiatypeRegular',
+  fontSize: 15,
+  fontStyle: 'normal',
+  fontWeight: 400,
+  lineHeight: 20,
+};
+const caption02 = {
+  fontFamily: 'DiatypeRegular',
+  fontSize: 13,
+  fontStyle: 'normal',
+  fontWeight: 400,
+  lineHeight: 16,
+};
+
```

Comment:
These values we should be able to pull directly from tokens right?

Reply:
yea but currently everything is in rem and web specific (rem is non-existant in react native). I would need to create base values for these, and then we would modify those for web and react native.

Reply:
i've got this design system working, now moving it to a separate package

Reply:
We have another task to fix the typography just created here: https://github.com/leather-wallet/extension/issues/4982 

As part of that lets alter it to make it more React Native friendly and use `px` over `rem` if it means we can share it

Reply:
I'm sure there are conversion tool etc we can use where it's just a case of looping props etc. 

Might end up with some funky theme definitions, but worth it if coming from a single source of truth